src/components/NavMenu.js

```diff
@@ -76,6 +79,13 @@ class NavMenu extends Component {
       });
     });
 
+    let logout = () => {
+      clearAuthToken();
+      redirectToLogin(false);
+    }
+
+    const logout_tooltip = (<Tooltip id='logout' className='tooltip'>{translate('logout')}</Tooltip>);
```

Comment:
I don't see the 'tooltip' class defined anywhere. Do we need it?

Reply:
I copied that from some other code, I'll try without it